hack/shoot_gcp.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: core.gardener.cloud/v1beta1
 kind: Shoot
 metadata:
   name: ${GARDENER_CLUSTER_NAME}
+  annotations:
+    dashboard.garden.sapcloud.io/no-hibernation-schedule: 'true'
```

Comment:
If a cluster cannot get deprovisioned and just stays. Then it will be good if it hibernates till someone realizes it and will manually delete it. -> Are you sure to disable it?

Reply:
hibernation was disabled all the time (on purpose but still with broken configuration). this annotation just removes a ui hint that tells you: "this cluster has no hibernation schedule enabled... are you sure about this?"

Reply:
Then for the gardener integration jobs at least it is enabled and it was helpful in the past as it saved many for orphaned instances, why not trying to do it here as well?